tensorly/backend/mxnet_backend.py

```diff
@@ -207,6 +218,17 @@ def partial_svd(matrix, n_eigenvecs=None):
         U, S, V = U[:, ::-1], S[::-1], V[:, ::-1]
         return tensor(U, **ctx), tensor(S, **ctx), tensor(V.T, **ctx)
 
+def qr(matrix, **kwds):
+    # NOTE - should be replaced with geqrf when available
+    try:
+        Q, R = map(nd.transpose, nd.linalg.gelqf(matrix.T))
```

Comment:
Personally I'd rather have 
```
Q, R = nd.linalg.gelqf(matrix.T)
return Q.T R.T
```
I find it makes what the code does more evident to the reader (generally I tend to not use map unless I have to)..

Reply:
Sure. Good point.